utils/tools.js

```diff
@@ -7,8 +7,7 @@ const dUTF = data => forge.util.decodeUtf8(data);
 const eUTF = data => forge.util.encodeUtf8(data);
 
 const generateKeyPair = () => {
-  const rsa = forge.pki.rsa;
-  const keypair = rsa.generateKeyPair({ bits: 2048, e: 0x10001 });
+  const keypair = forge.pki.rsa.generateKeyPair({ bits: 2048, e: 0x10001 });
   const pk = e64(forge.pki.publicKeyToPem(keypair.publicKey));
```

Comment:
@HexRabbit 
Since the `generateKeyPair` function is not used, is it okay to remove it and move the `createKeyPair` into the `tools.js`? 🤔

Reply:
Yes. For sure. 👍